test/controllers/tasks_controller_test.rb

```diff
@@ -90,39 +91,119 @@
   # Unskip and complete these tests for Wave 3
   describe "edit" do
     it "can get the edit page for an existing task" do
-      skip
-      # Your code here
+      # successfully goes to edit page
+      # Arrange
+
+      # Act
+      get edit_task_path(task)
+      # Assert
+      must_respond_with :success
     end
 
     it "will respond with redirect when attempting to edit a nonexistant task" do
-      skip
-      # Your code here
+      # successfully rediredts
+      # Arrange
+      # Act
+      get edit_task_path(3456)
```

Comment:
theoretically this could be a valid ID.  Instead I suggest using -1 or "invalid id" as the ID for this test.